fixieai/agents-ts/.eslintrc.cjs

```diff
@@ -6,7 +8,7 @@ module.exports = {
   ],
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    project: 'tsconfig.json',
+    project: path.join(__dirname__, 'tsconfig.json'),
```

Comment:
This fixes an issue where VSCode was looking in the project root for the `tsconfig.json`, which isn't the right place.